autoware_launch/config/system/system_diagnostic_monitor/planning.yaml

```diff
@@ -1,90 +1,203 @@
 units:
-  - path: /autoware/planning
-    type: short-circuit-and
```

Comment:
The short-circuit-and type means that if an element has an error, the diagnostics for the following elements should not be evaluated. This does not seem to be maintained in the new configuration.